contracts/profiles/AddrResolver.sol

```diff
@@ -1,23 +1,25 @@
-pragma solidity ^0.5.0;
+pragma solidity ^0.5.8;
 
 import "../ResolverBase.sol";
 
 contract AddrResolver is ResolverBase {
     bytes4 constant private ADDR_INTERFACE_ID = 0x3b3b57de;
+    bytes4 constant private ADDRESS_INTERFACE_ID = 0xf1cb7e06;
+    uint constant private COIN_TYPE_ETH = 60;
 
     event AddrChanged(bytes32 indexed node, address a);
+    event AddressChanged(bytes32 indexed node, uint coinType, bytes newAddress);
 
-    mapping(bytes32=>address) addresses;
+    mapping(bytes32=>mapping(uint=>bytes)) _addresses;
```

Comment:
Why is ETH 60?

Reply:
Addr id are to follow SLIP-0044 : https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
@amxx perfect thanks